frontend/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:18-alpine
 
+ENV REACT_APP_ENV production
```

Comment:
This may cause some issues when ppl want to test in their local machines. Is there any way to configure this flag in `docker-compose`? Say `docker compose up --env production`.

Reply:
I can look into it. There's definitely a way to pass a parameter to set ENV from the docker compose file, though. 

Alternatively we can set it to production in the main branch and development in the dev branch.

Reply:
Why do we have this change: [b5c00df](https://github.com/collebrusco/ECE-461L-Orange/pull/55/commits/b5c00dfc160f0d6438139deafce469b4b6f43ac2)

Reply:
The variable wasn't being passed down properly. I can try reverting and see what happens.

Reply:
Ok nevermind it works properly... I must have forgotten to fully rebuild. We can revert.